api/v1alpha1/mariadb_types.go

```diff
@@ -70,6 +70,9 @@ func (p *PodDisruptionBudget) Validate() error {
 type ServiceTemplate struct {
 	Type        corev1.ServiceType `json:"type,omitempty"`
 	Annotations map[string]string  `json:"annotations,omitempty"`
+	externalTrafficPolicy string `json:"externalTrafficPolicy,omitempty"`
+	loadBalancerSourceRanges []string `json:"loadBalancerSourceRanges,omitempty"`
+	loadBalancerIp string `json:"loadBalancerIp,omitempty"`
```

Comment:
This fields need to start with capital letter to be exported. Could you update them please